src/components/list-header/list-header.scss

```diff
@@ -0,0 +1,12 @@
+.list-header {
+    background: #1C1080;
+    color: #FFFFFF;
+    display: inline-block;
+    font-family: 'Titillium Web', sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 30px;
+    padding: 15px 23px 19px 26px;
+    width: 673px;
```

Comment:
We should have it fluid. `width: 100%` and when used it will be wrapped in different containers.

Reply:
It makes sense, indeed, thanks for pointing it out; fixed it.